kafka-rest/src/test/java/io/confluent/kafkarest/integration/ClusterTestHarness.java

```diff
@@ -419,6 +426,23 @@ protected final void createTopic(String topicName, int numPartitions, short repl
     }
   }
 
+  protected final void setTopicConfig(String topicName, String configName, String value) {
+    Properties properties = restConfig.getAdminProperties();
+    properties.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, brokerList);
+    AdminClient adminClient = AdminClient.create(properties);
+
+    AlterConfigsResult result =
+        adminClient.incrementalAlterConfigs(
+            singletonMap(
+                new ConfigResource(ConfigResource.Type.TOPIC,topicName),
+                singletonList(new AlterConfigOp(new ConfigEntry(configName,value), AlterConfigOp.OpType.SET))));
+    try {
+      result.all().get();
+    } catch (InterruptedException | ExecutionException e) {
+      Assert.fail(String.format("Failed to alter topic config: %s", e.getMessage()));
```

Comment:
nit: I'd also suggest including the topic name and maybe the config name here.

Reply:
fixed